testing/testing_lib.sh

```diff
@@ -46,12 +46,36 @@ function run_jobmanager() {
     local image_tag="$(image_tag "$dockerfile")"
     local image_name="$(image_name "$image_tag")"
 
+    echo >&2 "Running ${image_tag} jobmanager..."
```

Comment:
I would prefer to call this `Starting` or something different of `Running` because it will be easier to find specially when looking at the raw logs because docker logs build steps as 'Running'.

Reply:
Sure, will change.